ewatercycle/util.py

```diff
@@ -71,12 +72,12 @@ def lat_lon_boundingbox_to_variable_indices(model, variable, latMin, latMax, lon
     """
     # get shape of model grid and lat-lon coordinates of grid
     shape = model.get_grid_shape(model.get_var_grid(variable))
-    latModel = model.get_grid_x(model.get_var_grid(variable))
-    lonModel = model.get_grid_y(model.get_var_grid(variable))
-    nx = len(latModel)
+    lat_model = model.get_grid_x(model.get_var_grid(variable))
+    lon_model = model.get_grid_y(model.get_var_grid(variable))
+    nx = len(lat_model)
 
-    idx = [i for i,v in enumerate(latModel) if ((v > latMin) and (v < latMax))]
-    idy = [i for i,v in enumerate(lonModel) if ((v > lonMin) and (v < lonMax))]
+    idx = [i for i,v in enumerate(lat_model) if ((v > latMin) and (v < latMax))]
```

Comment:
```suggestion
    idx = [i for i,v in enumerate(lat_model) if ((v > lat_min) and (v < lat_max))]
```

Reply:
should these variable names not also be adjusted?

Reply:
Hmm, pycharm did not complain about those, but yes let's also adjust those vars.